front/preface.tex

```diff
@@ -17,4 +17,4 @@
 
 \hfill \break
 
-Sondre Lefsaker and Håvard Wormdal Høiby \textbox{\hfill 12.06.2015}
```

Comment:
Den er aligna til høyre for navnene, men den er ikke right-aligned ved margen.

Er det [denne](http://tex.stackexchange.com/questions/55472/how-to-make-text-aligned-left-center-right-in-the-same-line) posten du har sett på?
Isåfall tror jeg kanskje du har glemt å definere textbox-kommandoen?